src/page/public/Video.jsx

```diff
@@ -76,11 +83,21 @@ export const Video = () => {
                   onClick={() => dispatch(postLikedVideos(video))}
                 />
               )}
-              <RiVideoAddFill
-                size={30}
-                className="cursor-pointer"
-                title="Add to Watch Later"
-              />
+              {isInWatchLater ? (
```

Comment:
`isInWaterLater` sounds a little weird. Can have better naming for this variable. 
Rest of the code looks well structed

Reply:
will change to `videoIsInWatchLater`